src/renderer/main.js

```diff
@@ -19,7 +19,19 @@ let fs = require('fs');
 let path = require('path');
 let me = fs.readFileSync(path.join(__dirname, 'api-decorator.js'), 'utf8');
 
-let jsAdapter = fs.readFileSync(path.join(process.resourcesPath, 'adapter.asar', 'openfin-desktop.js'), 'utf8');
+// check resources/adapter/openfin-desktop.js then
+// resources/adapter.asar/openfin-desktop.js
+// for ease of developement
+let jsAdapter = '';
+const searchPaths = ['adapter', 'adapter.asar'];
```

Comment:
I would prefer if this is done via a flag,  we would want to prevent this on non-dev scenarios.

Reply:
The core has the same behavior without a flag: https://github.com/openfin/runtime/blob/develop/lib/browser/init.js#L115

Reply:
Super edge case... if someone is running in enforced signed assets mode and happens to have that js file sitting there, it will fail first even though a valid asset is present right? Again, I don't think I really care that much...